sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -161,6 +165,13 @@ def rank_error(msg):
         if not page_token:
           break
 
+      if duration:
+        passed_secs = time.time() - start_secs
+        if duration_secs > passed_secs:
+          logging.info('Timing out on waiting for job %s after %d seconds',
```

Comment:
<!--new_thread; commit:5b7323ad9e10a5693ea41a239c8a675139b977de; resolved:0-->
Should this be a warning?  Can you also add in the docstring documentation for what is returned when timed out?

Reply:
Done. Updated the docstring, note that nothing is returned.